### PR TITLE
OKTA-431895: Add API_ACCESS_MANAGEMENT_ADMIN to role types

### PR DESCRIPTION
**Pull request by alimohammadi-okta**
## Description:
- **What's changed?** Update Subscriptions-Role API to support bot API_ACCESS_MANAGEMENT_ADMIN instead & API_ADMIN
- **Is this PR related to a Monolith release?** Yes. 2022.10.0

### Resolves:

[OKTA-431895](https://oktainc.atlassian.net/browse/OKTA-431895)

### Reviewers:
@williamngo-okta @rickyyang-okta @okta/dda 

## Screenshot:
Screen Shot 2022-08-30 at 3 37 28 PM
0f-18a03f75bed8.png)

### CODE REVIEW

**Review by rickyyang-okta — APPROVED** [review]

**Review by rickyyang-okta — COMMENTED** [review]
Should we leave a disclaimer or note or something for the overlap in Label?

**Comment by alimohammadi-okta** [comment]
> Should we leave a disclaimer or note or something for the overlap in Label?

Merged two roles into one line to avoid misunderstanding (https://github.com/okta/okta-developer-docs/commit/3c1a00f37a07a7993b5eb03934d73b0145239bab)

File: packages/@okta/vuepress-site/docs/reference/api/admin-notifications/index.md [hunk 1]
```diff
@@ -944,7 +944,7 @@ Refer to the [product documentation](https://help.okta.com/okta_help.htm?id=ext_
 
 | Role type                     | Label                               |
 | :---------------------------- | :---------------------------------- |
-| `API_ADMIN`                   | API Access Management Administrator |
+| `API_ADMIN` (or `API_ACCESS_MANAGEMENT_ADMIN`) | API Access Management Administrator |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/admin-notifications/index.md:947]:
```suggestion
| `API_ADMIN` or `API_ACCESS_MANAGEMENT_ADMIN` | API Access Management Administrator |
```

**Review by susanharper-okta — COMMENTED** [review]
one comment.

File: packages/@okta/vuepress-site/docs/reference/api/admin-notifications/index.md (hunk 1 above)
Comment (alimohammadi-okta) [reply packages/@okta/vuepress-site/docs/reference/api/admin-notifications/index.md]: Addressed in https://github.com/okta/okta-developer-docs/commit/d9c59f41591f3b39c75ecb0e2354db187cb87a99

**Comment by alimohammadi-okta** [comment]
This is should be merged into `Monthly Release: 2022.10.0 CF`

**Review by williamngo-okta — APPROVED** [review]

**Pull request merged**